edgelet/version.txt

```diff
@@ -1 +1 @@
-1.0.9.2
\ No newline at end of file
+1.0.9.3
```

Comment:
Should this match edge agent/hub version?

Reply:
Nope, we didn't have an iotedged release for 1.0.9.3.  Now the edgelet is one behind.

Reply:
what have we done previously? Let's follow the same pattern.

Reply:
Looks like we did release edgelet for 1.0.9.3 - https://github.com/Azure/azure-iotedge/releases/tag/1.0.9.3